inc/languages/english/admin/config_banning.lang.php

```diff
@@ -34,7 +34,9 @@
 $l['ban_ip_address'] = "Ban IP Address";
 
 $l['error_missing_ban_input'] = "You did not enter a value to ban.";
-$l['error_invalid_filter'] = "The specified filter does not exist.";
+$l['error_invalid_filter'] = "The specified filter is not valid.";
+$l['error_filter_not_found'] = "The specified filter does not exist.";
+$l['error_filter_already_banned'] = "The filter you entered is already banned.";
```

Comment:
This line duplicates the line below. Suggestion: remove it.

Reply:
Whoops, done!